data/sql/updates/pending_db_world/rev_1748010106966597790.sql

```diff
@@ -0,0 +1,6241 @@
+--
+SET @CGUID := 153321; -- creature start guid
+SET @GGUID := 83038; -- gameobject start guid
+
+DELETE FROM `command` WHERE `name` IN (
+    'worldstate scourgeinvasion show',
+    'worldstate scourgeinvasion state',
+    'worldstate scourgeinvasion battleswon',
+    'worldstate scourgeinvasion startzone'
+);
+
+INSERT INTO `command` (`name`, `security`, `help`) VALUES
+('worldstate scourgeinvasion show',       3, 'Syntax: .worldstate scourgeinvasion show\nDisplays the current status of the Scourge Invasion.'),
+('worldstate scourgeinvasion state',      3, 'Syntax: .worldstate scourgeinvasion state <value>\nSets the Scourge Invasion state.\nValid values:\n0: Disabled\n1: Enabled'),
+('worldstate scourgeinvasion battleswon', 3, 'Syntax: .worldstate scourgeinvasion battleswon <value>\nAdjusts the Scourge Invasion battles won count by <value> (can be negative).'),
+('worldstate scourgeinvasion startzone',  3, 'Syntax: .worldstate scourgeinvasion startzone <id>\nStarts a Scourge Invasion event in the zone specified by <id>.\nValid zone IDs: 0-7.');
+
+SET
+@scourge_invasion            = 17,
+@scourge_boss_in_instance    = 120,
+@scourge_winterspring        = 121,
+@scourge_tanaris             = 122,
+@scourge_azshara             = 123,
+@scourge_blasted_lands       = 124,
+@scourge_eastern_plaguelands = 125,
+@scourge_burning_steppes     = 126,
+@scourge_50_invasions_done   = 127,
+@scourge_100_invasions_done  = 128,
+@scourge_150_invasions_done  = 129,
+@scourge_invasions_done      = 130;
+
+DELETE FROM `game_event` WHERE `eventEntry` IN (
+    @scourge_boss_in_instance,  @scourge_winterspring, @scourge_tanaris, @scourge_azshara,
+    @scourge_blasted_lands,     @scourge_eastern_plaguelands, @scourge_burning_steppes,
+    @scourge_50_invasions_done, @scourge_100_invasions_done, @scourge_150_invasions_done, @scourge_invasions_done
+);
+SET @GAMEEVENT_INTERNAL = 5; -- never handled in GameEventMgr update
```

Comment:
Put it either with the SET above (in the same block) or under it but above the delete